### DISCUSSION TITLE
Im trying to make a server for the 1.21 in raspberry pi 4 b and i have this error

### DISCUSSION THREAD

**Discussion by Aimar27** [itzg/docker-minecraft-server#2908] [Q&A]
Im using this code: 
docker run -e EULA=TRUE -it -v dataminecraft:/minecraft -p 25565:25565 -p 19132:19132/udp -p 19132:19132 --restart unless-stopped itzg/minecraft-server:java17

And this is the error: 
LinkageError occurred while loading main class net.minecraft.bundler.Main
        java.lang.UnsupportedClassVersionError: net/minecraft/bundler/Main has been compiled by a more recent version of the Java Runtime (class file version 65.0), this version of the Java Runtime only recognizes class file versions up to 61.0

Can someone help me?

**Comment by itzg** [comment]
https://docker-minecraft-server.readthedocs.io/en/latest/versions/java/#class-file-version-650

**Reply by Aimar27** [reply]
Thank you itzg but the raspberry pi 4 is arm v8
If I try with java 21 I got these mesage

Unable to find image 'itzg/minecraft-server:java21' locally
java21: Pulling from itzg/minecraft-server
docker: no matching manifest for linux/arm/v8 in the manifest list entries.

**Reply by itzg** [reply]
Im afraid I am constrained by the upstream Java distribution images all of which seem to no longer support arm/v8 😒

**Reply by Aimar27** [reply]
Ok thanks :(
